### ISSUE TITLE
fix(accessibility): main landmarks, presentational and labeled icons, no empty headings

### ISSUE THREAD

**Issue by billhimmelsbach** [cfpb/sbl-frontend#356]
We need to implement the small accessibility changes that are possible with the accessibility changes in `design-system-react`:
- [[Hero] Allow hero headings to be `<h1>`, `<h2>`, etc. instead of unstyled headings wrapped in `<p>` tags](https://github.com/cfpb/design-system-react/issues/328)
- [[Icon] Make icons more accessible](https://github.com/cfpb/design-system-react/issues/324) 

## Current behavior
- some pages lack accessibility features

## Expected behavior
- add missing `main` landmarks on a few pages that are missing it
- make `Icons` presentational when the content they are next to already provide the meaning
- add `ariaLabel`s to `Icons` that contain content that are necessary for understanding the surrounding context
- Ensure that there are no empty `Heading`s

## Notes
- these changes are limited to the shared lending platform for now
